frontend/packages/policy-editor/src/PolicyEditor.tsx

```diff
@@ -96,10 +90,7 @@ export const PolicyEditor = ({
         <div className={classes.alertWrapper}>
           <PolicyEditorAlert />
         </div>
-        <PolicyCardRules showErrorsOnAllRulesAboveNew={showErrorsOnAllRulesAboveNew} />
-        <div className={classes.addCardButtonWrapper}>
-          <AddPolicyRuleButton onClick={handleClickAddButton} />
-        </div>
+        {usageType === 'app' ? <PolicyEditorTabs /> : <PolicyRulesEditor />}
```

Comment:
I suggest adding a component at the bottom of the file to increase readability 😄 

```tsx
...
               <PolicyEditorUsageContent usageType={usageType} />
...

type PolicyEditorUsageContentProps = {
   usageType: PolicyEditorUsage
}

function PolicyEditorUsageContent({ usageType }: PolicyEditorUsageContentProps): ReactElement {
   if (usageType === 'app') {
      return <PolicyEditorTabs />
   }
   return <PolicyRulesEditor />
}
```

Reply:
Thanks for the suggestion, I committed your code snippet 😊 

For future reference, this type of comment could be prefixed with "nit"/"nitpick" - as in, small thing that does not affect the code/pr outcome, but nice to consider 😊